app/forms/additional_property_details_form.rb

```diff
@@ -7,7 +7,8 @@ class AdditionalPropertyDetailsForm
   ITEMS_SESSION_KEY = "additional_properties".freeze
   ITEM_MODEL = AdditionalPropertyModel
   ATTRIBUTES = %i[additional_properties].freeze
-  alias_attribute :additional_properties, :items
+  alias_method :additional_properties, :items
+  alias_method :additional_properties=, :items=
```

Comment:
This took me quite a bit of reading to understand. 

Just leaving this here in case it comes in handy in the future. They may look to add in [alias_association](https://github.com/rails/rails/pull/49801) into a future version of rails, which might be useful.